docs-source/content/userguide/managing-domains/model-in-image/runtime-updates.md

```diff
@@ -0,0 +1,325 @@
+
+This example references a database running in the `default` namespace that is accessed with the URL `oracle-db.default.svc.cluster.local:1521/devpdb.k8s`, user `sys as dba`, and password `Oradoc_db1`. Note that you can still add this data source even if there's no database running at this location.
+
+{{% notice note %}} If you already have a WDT config map deployed for your running domain, as is true for the [Model in Image]({{< relref "/samples/simple/domains/model-in-image/_index.md" >}}) sample, then you should ensure that it includes any other needed files that were in the original WDT config map, in addition to the new data source YAML file. For example, the Model in Image sample puts file(s) in a directory that it uses to stage its config map in `$WORKDIR/wdtconfigmap`.
+{{% /notice %}}
+
+
+  ```
+  #!/bin/bash
+
+  set -eu
+
+  # Assume SAMPLEDIR references the 'kubernetes/samples/scripts/create-weblogic-domain/model-in-image' directory within the operator source.
+
+  # Assume the DB is running in the following namespace
+
+  DB_NAMESPACE="default"
+
+  # Assume DOMAIN_UID is the following
+
+  DOMAIN_UID=sample-domain1
+
+  # Assume DOMAIN_NAMESPACE is the following
+
+  DOMAIN_NAMESPACE=sample-domain1-ns
+
+  # Assume the DB URL and password are defined in a secret as follows
+
+  $SAMPLEDIR/create_secret.sh \
+    -n ${DOMAIN_NAMESPACE} \
+    -s ${DOMAIN_UID}-new-db-access-secret \
+    -l password=Oradoc_db1 \
+    -l url=oracle-db.${DB_NAMESPACE}.svc.cluster.local:1521/devpdb.k8s
+
+  # Assume WORKDIR is your working directory
+
+  cd $WORKDIR
+
+  # Create a WDT configmap with the datasource WDT yaml snippet
+
+  mkdir -p ./wdtconfigmap
+
+  cat << EOF > wdtconfigmap/datasource.yaml
+  resources:
+    JDBCSystemResource:
+      mynewdatasource:
+        Target: 'cluster-1'
+        JdbcResource:
+          JDBCDataSourceParams:
+            JNDIName: [
+              jdbc/generic2,
+              jdbc/special2
+            ]
+            GlobalTransactionsProtocol: TwoPhaseCommit
+          JDBCDriverParams:
+            DriverName: oracle.jdbc.xa.client.OracleXADataSource
+            URL:               '@@SECRET:@@ENV:DOMAIN_UID@@-new-db-access-secret/url@@'
+            PasswordEncrypted: '@@SECRET:@@ENV:DOMAIN_UID@@-new-db-access-secret/password@@'
+            Properties:
+              user:
+                Value: 'sys as sysdba'
+              oracle.net.CONNECT_TIMEOUT:
+                Value: 5000
+              oracle.jdbc.ReadTimeout:
+                Value: 30000
+          JDBCConnectionPoolParams:
+              InitialCapacity: 0
+              MaxCapacity: 1                   # This is a comment
```

Comment:
What is "This is a comment" for? :)

Reply:
Assuming you're serious:  it's to teach folks how one can put comments in a model file.

Reply:
I was being serious!  Okay, then I'd prefer: "# Include comments like this"

Reply:
Tom AI: Remove fancy macro-within-macro, to be more clear.   Tom AI: Replace "/" with ":" that's a typo.

Reply:
done